backend/src/main/java/com/woowacourse/naepyeon/service/TeamService.java

```diff
@@ -188,9 +195,20 @@ public boolean isJoinedMember(final Long memberId, final Long teamId) {
                 .anyMatch(team -> team.getId().equals(teamId));
     }
 
-    public String createInviteToken(final Long teamId) {
-        validateExistTeam(teamId);
-        return inviteTokenProvider.createInviteToken(teamId);
+    @Transactional
+    public String createInviteCode(final Long teamId) {
+        final Team team = teamRepository.findById(teamId)
+                .orElseThrow(() -> new NotFoundTeamException(teamId));
+        final InviteCode inviteCode = team.createInviteCode(
+                () -> SecureRandomStringUtils.createRandomAlphanumericSecure(10)
```

Comment:
```suggestion
                () -> SecureRandomStringUtils.createRandomAlphanumericSecure(INVITE_CODE_LENGTH)
```
위와 같이 상수화를 하는 건 어떤가요?

Reply:
상수화 하는게 좋아보이네요!